src/TextEngine/GameConsole.cpp

```diff
@@ -367,14 +367,12 @@ void CGameConsole::DrawW(
 void CGameConsole::SaveConsoleMessage()
 {
     DEBUG_TRACE_FUNCTION;
-    if (m_ConsoleStack[m_ConsoleSelectedIndex % MAX_CONSOLE_STACK_SIZE] != Text)
+
+    m_ConsoleStack[m_ConsoleStackCount % MAX_CONSOLE_STACK_SIZE] = Text;
```

Comment:
Is this right? We will be able to save repeated message? I'll need to check on the original client too.

Reply:
yes all message in original client are save even repeated one ....

Reply:
but if you want we can keep these as a feature of CrossUO ...

Reply:
I don't think it is worth as a feature anyway, lets be more parity than custom.